app/javascript/packs/document-capture-welcome.ts

```diff
@@ -0,0 +1,67 @@
+import { hasCamera, isCameraCapableMobile } from '@18f/identity-device';
```

Comment:
(You'll want to compare this file with the .js version above, as I converted it to Typescript)